src/os/bluestore/BlueFS.h

```diff
@@ -338,6 +341,17 @@ class BlueFS {
 			  uint64_t jump_to = 0);
   uint64_t _estimate_log_size();
   bool _should_compact_log();
+  uint64_t _default_bluefs_alloc_size(uint64_t bluestore_min_alloc_size) {
+    uint64_t min_allowed_size = 16384U; // same as bluestore_min_alloc_size_ssd
```

Comment:
this can be removed once we have an assertion that prevents superblock from overwrite